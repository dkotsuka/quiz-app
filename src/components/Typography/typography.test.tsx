import React from 'react'
import { Typography } from '.'
import { create } from 'react-test-renderer';

describe("Typography", () => {
    const typoH1 = create(<Typography.H1>H1 Text</Typography.H1>)
    const typoH2 = create(<Typography.H2>h2 Text</Typography.H2>)
    const typoBody = create(<Typography.Body>Body Text</Typography.Body>)
    const typoSmall = create(<Typography.Small>Small Text</Typography.Small>)

    it("maches snapshot", () => {
        expect(typoH1).toMatchSnapshot()
        expect(typoH2).toMatchSnapshot()
        expect(typoBody).toMatchSnapshot()
        expect(typoSmall).toMatchSnapshot()
    })
})