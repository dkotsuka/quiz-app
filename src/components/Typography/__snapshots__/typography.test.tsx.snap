// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Typography maches snapshot 1`] = `
<Text
  style={
    Array [
      Object {
        "fontFamily": "Roboto Bold",
        "fontSize": 20,
      },
    ]
  }
>
  H1 Text
</Text>
`;

exports[`Typography maches snapshot 2`] = `
<Text
  style={
    Array [
      Object {
        "fontFamily": "Roboto Bold",
        "fontSize": 16,
      },
    ]
  }
>
  h2 Text
</Text>
`;

exports[`Typography maches snapshot 3`] = `
<Text
  style={
    Array [
      Object {
        "fontFamily": "Roboto",
        "fontSize": 14,
      },
    ]
  }
>
  Body Text
</Text>
`;

exports[`Typography maches snapshot 4`] = `
<Text
  style={
    Array [
      Object {
        "fontFamily": "Roboto",
        "fontSize": 12,
      },
    ]
  }
>
  Small Text
</Text>
`;
